crypto/err/err_local.h

```
@@ -76,8 +79,11 @@ static ossl_inline void err_set_data(ERR_STATE *es, size_t i,
     es->err_data_flags[i] = flags;
 }
 
-static ossl_inline void err_clear(ERR_STATE *es, size_t i, int deall)
+static ossl_inline void err_clear(ERR_STATE *es, int i, int deall)
```

Why this change?

---

I changed this simply because everywhere else in `crypto/err/`, the index `i` in the error queue is simply an `int`,
and I find it confusing that when calling this function it is silently converted to `size_t`.

---

It is not int in the calls above.

---

I reverted that change.